ibis/backends/impala/client.py

```diff
@@ -223,7 +224,7 @@ def execute(self, stmt):
 
     def _wait_synchronous(self):
         # Wait to finish, but cancel if KeyboardInterrupt
-        from impala.hiveserver2 import OperationalError
+        from .hiveserver2 import OperationalError
```

Comment:
I think this `impala` is the `impyla` package one, quite misleading, I know.

Reply:
thanks for info.  in compat.py there is an import dbapi as impyla but i dont see dbapi.py anywhere.  Is that what you are referring to?

Reply:
There is an external package (i.e. `pip install impyla`) that provides an impala modules (i.e. `import impala`). That's unrelated to Ibis. Then Ibis has a `ibis.impala` module (now moved to `ibis.backends.impala` in this PR), and also, the ibis module has a `ibis.impala` attribute, that is an alias of `ibis.backends.impala` now (you updated the import in `ibis/__init__.py`).

Very confusing, surely not great, but for now just revert the changes here and in the rest of places I commented. Since you thought this was the `impala` in `ibis.impala`, but it's the `impala` in the `impyla` package.

Reply:
okay, i think i get it.  i think i fixed it.  also i highlighted another area that i think is impacted by this (errors highlighted below).

Reply:
im still getting no module not found error for impala, but i dont see impyla in the environment setup. does it need to be added so we can import it?

Reply:
If you mean locally, we have every backend dependencies in `ci/deps/<backend>.yml`.

Reply:
Ok, i see it there. I just didnt see it in the github actions setup environment step or the environment.yml (so it isnt resolving locally either since my conda env is based on that file).

Im a bit new to building packages so im going to look into it more.

Reply:
I saw on another pull request there were ci failures due to ibis dependencies not working. Could it be related?